go/store/types/edits/async_sorted_edits.go

```diff
@@ -173,6 +176,12 @@ func (ase *AsyncSortedEdits) FinishedEditing() (types.EditProvider, error) {
 	return ase.Iterator(), nil
 }
 
+func (ase *AsyncSortedEdits) Close() {
+	if !ase.closed {
```

Comment:
There's a race here. You should add a comment to this method describing what it does, and note that it isn't thread safe, and must be synchronized externally.